ignite/metrics/loss.py

```diff
@@ -42,35 +42,32 @@ class Loss(Metric):
         and the Loss metric using an ``evaluator`` created with
         :meth:`~ignite.engine.create_supervised_evaluator` method.
 
-        .. code-block:: python
+        .. testsetup:: *
+        
+            default_trainer = get_default_trainer()
 
-            import torch
-            import torch.nn as nn
-            from torch.nn.functional import nll_loss
+        .. testcode::
 
-            from ignite.metrics import Accuracy, Loss
-            from ignite.engine import create_supervised_evaluator
-
-            model = ...
+            model = default_model
 
             criterion = nll_loss
+            c_kwargs = {"reduction": "sum"}
 
-            metrics = {
-                "Accuracy": Accuracy(),
-                "Loss": Loss(criterion)
-            }
+            def output_transform(output):
```

Comment:
Here, this is useless.

However, it could be great to add a very concise example showing how handle the multi output case mentioned in the doc, what do you think ?